### ISSUE TITLE
amp-image-lightbox hard to close

### ISSUE THREAD

**Issue by samuelcolvin** [ampproject/amphtml#15198]
## What's the issue?

The `amp-image-lightbox` modal can be hard to close, especially when the aspect ratio of the image matches the screen. I know "back" works, but users can be very dumb, also people are rightly unsure about click back as it might take them away from that page.

Possible solutions:
* add an explicit "close" button on top of the image
* closing the modal upon clicking/tapping the image, or allow this via `on`
* allow css to pad the image (clicking/tapping around the image closes the modal) - this would also improve aesthetic

## How do we reproduce the issue?

1. go to https://ampbyexample.com/components/amp-image-lightbox/
2. set your browser window to be roughly square
3. open the picture of the first dog
4. you'll find it hard to close the modal without using the back button

## What browsers are affected?

all/na, desktop and mobile.

## Which AMP version is affected?

`Powered by AMP ⚡ HTML – Version 1525461683159 http://localhost:8000/us/blog/forum`

Not new AFAIK.

**Comment by cathyxz** [comment]
Does `<amp-lightbox-gallery>` work for you? 
https://ampbyexample.com/components/amp-lightbox-gallery/
It does have a close button. =)

**Comment by samuelcolvin** [comment]
Thanks for the quick response.

The close button there works well, however:
* the back button doesn't work with `amp-lightbox-gallery`
* it shows the left and right arrows and the "gallery" button in the top left even if there's only one image on the page.
* it's still an experiment so I can't use it in production :-).

In short, for now it's not usable.

**Comment by jaygray0919** [comment]
We have a similar problem, as seen here:

  https://afdsi.org/test/amp_list_carousel_fixed_height/

this behavior changed within the last several weeks.

we raised this and a similar issue here:

https://amphtml.slack.com/archives/C9HPA6HGB/p1525687102000141

**Comment by jaygray0919** [comment]
OK, here we use amp-lightbox-gallery
  https://ampb.in/#-LCAnotP1ZK-neodjfaK
Lightbox fails (we have AMP experiment set to `on` for amp-lightbox-playground)

But, when we use AMP Playground, the lightbox works perfectly. But we can't save the instance. You can copy the AMPbin code to AMP Playground and see the result

In general, nice progress - thank you

@cathyxz Cathy, is it possible to have layout=fixed-height (to show a tall image that has been scaled to fit the carousel) but use 'responsive' for the lightbox?

If the answer is no, we're good with the new amp-lightbox-gallery and will work within its limits.

**Comment by cathyxz** [comment]
Yup, it should be. You found a bug. I was trying to be efficient with the `width` and `height` attributes, but didn't handle the situation where `width` can be `auto`. I'm patching it here: https://github.com/ampproject/amphtml/pull/15206. `<amp-lightbox-gallery>` should launch within the next two releases.

**Comment by cathyxz** [comment]
Re: @samuelcolvin 
TLDR is we are not taking new FRs for `<amp-image-lightbox>` because our goal is for `<amp-lightbox-gallery>` to deprecate it. 
> - the back button doesn't work with amp-lightbox-gallery
> - it shows the left and right arrows and the "gallery" button in the top left even if there's only one image on the page

Both of these are reasonable FRs for `<amp-lightbox-gallery>`, which will launch with #15183.

**Comment by samuelcolvin** [comment]
Thanks @cathyxz, lightbox-gallery looks like the solution.
